src/header.ts

```diff
@@ -82,6 +85,20 @@ export interface HeaderConfig {
    */
   titleLimit?: number;
 
+  /**
+   * The orthogonal offset in pixels by which to displace the title from its position along the edge of the chart.
+   *
+   * __Default value:__ `10`, indicating no limit
+   */
+  titleOffset?: number;
```

Comment:
A similar thing is called titlePadding in Axis.

Reply:
Done